Enterprise-CMCS/macpro-mdct-mfp
# [MDCT-2964 + 2965] Admin Banner Handler Validation + Zustand Integration

### Description

This PR adds validation to the Admin Banner create API call and then further integrates Zustand into it. You should now see the full Admin Banner experience!

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/424dd941-ebc2-431b-b14a-b3d7a355280a

### Related ticket(s)


MDCT-2964
MDCT-2965

---
### How to test

Login as an admin (adminuser@test.com)
Open My Account through the top right dropdown
Open the Banner Editor
Create/Delete banners and visit the home page to see it live!

### Author checklist
